pep-perennial.rst

```diff
@@ -0,0 +1,161 @@
+PEP: XXX
+Title: Future 'manylinux' Platform Tags for Portable Linux Built Distributions
+Version: $Revision$
+Last-Modified: $Date$
+Author: Nathaniel J. Smith <njs@pobox.com>
+        Thomas Kluyver <thomas@kluyver.me.uk>
+BDFL-Delegate: Nick Coghlan <ncoghlan@gmail.com>
+Discussions-To: Distutils SIG <distutils-sig@python.org>
```

Comment:
Do you want to use distutils-sig, or switch to discourse?

Reply:
I'm only subscribed to specific threads on Discourse, so if someone starts a new one, I won't necessarily see it. So maybe +0.1 for distutils-sig. But I'm happy to change this if the consensus is that we're doing discussions on Discourse now.

Reply:
At least for this particular discussion, right now, it really does seem like we are doing the discussion on the Discourse thread. So I suggest that we change it to reflect the current reality.